crates/runtime/src/model/mod.rs

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2024 The Spice.ai OSS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+use arrow::record_batch::RecordBatch;
+use model_components::model::{Error as ModelError, Model};
+use std::result::Result;
+use std::sync::Arc;
+
+mod chat;
+mod embed;
+
+pub use chat::{try_to_chat_model, LLMModelStore};
+pub use embed::{try_to_embedding, EmbeddingModelStore};
+
+use crate::DataFusion;
+
+pub async fn run(m: &Model, df: Arc<DataFusion>) -> Result<RecordBatch, ModelError> {
+    match df
+        .ctx
+        .sql(
+            &(format!(
+                "select * from datafusion.public.{} order by ts asc",
```

Comment:
This looks incorrect now? It should at least be `spice.public.{}`

Reply:
Good catch, this is ML model code, not often executed.